dev/docker/Dockerfile.bigdl-cpu

```diff
@@ -27,7 +27,7 @@ RUN --mount=type=cache,target=/opt/conda/pkgs conda init bash && \
 COPY ./pyproject.toml .
 COPY ./MANIFEST.in .
 
-RUN mkdir ./finetune && mkdir ./inference
+RUN mkdir ./llm_on_ray
```

Comment:
You map the source code path to /root/llm-on-ray directory in workflow but create llm_on_ray here, what is the intention?

Reply:
Because llm_on_ray is package_dir, it needs to exist in the llm-on-ray repo directory when executing pip install -e, otherwise it will be report the following error,
```
Installing build dependencies ... done
Checking if build backend supports build_editable ... done
Getting requirements to build editable ... error
error: subprocess-exited-with-error

× Getting requirements to build editable did not run successfully.
│ exit code: 1
╰─> [10 lines of output]
    WARNING setuptools_scm._integration.setuptools pyproject.toml does not contain a tool.setuptools_scm section
    running egg_info
    creating llm_on_ray.egg-info
    writing llm_on_ray.egg-info/PKG-INFO
    writing dependency_links to llm_on_ray.egg-info/dependency_links.txt
    writing entry points to llm_on_ray.egg-info/entry_points.txt
    writing requirements to llm_on_ray.egg-info/requires.txt
    writing top-level names to llm_on_ray.egg-info/top_level.txt
    writing manifest file 'llm_on_ray.egg-info/SOURCES.txt'
    error: package directory 'llm_on_ray' does not exist
    [end of output]
```